components/skeleton/skeleton-styles.js

```diff
@@ -0,0 +1,30 @@
+import { css } from 'lit-element/lit-element.js';
+
+export const skeletonStyles = css`
+	@keyframes loadingPulse {
+		0% { background-color: var(--d2l-color-sylvite); }
+		50% { background-color: var(--d2l-color-regolith); }
+		75% { background-color: var(--d2l-color-sylvite); }
+		100% { background-color: var(--d2l-color-sylvite); }
+	}
+
```

Comment:
This is the core of the solution using an absolutely position before style to cover the component being styled.

Reply:
Oh, this is interesting idea and I like it.